### PR TITLE
Enable open-cluster-management observatorium-operator

### PR DESCRIPTION
**Pull request by clyang82**
Signed-off-by: clyang82 <chuyang@redhat.com>

### CODE REVIEW

File: ci-operator/config/open-cluster-management/observatorium-operator/open-cluster-management-observatorium-operator-release-2.1.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  namespace: open-cluster-management
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 1Gi
+tag_specification:
+  name: "4.4"
```
Comment (petr-muller) [inline ci-operator/config/open-cluster-management/observatorium-operator/open-cluster-management-observatorium-operator-release-2.1.yaml:20]: Huh are you sure you want to test on 4.4 and not something newer?

File: ci-operator/config/open-cluster-management/observatorium-operator/open-cluster-management-observatorium-operator-release-2.1.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+test_binary_build_commands: "true"
+tests:
+- as: test
+  commands: |
+    KUBECTL=oc
+    OPERATOR_IMAGE_NAME=${IMAGE_FORMAT//\/stable:\$\{component\}//pipeline:observatorium-operator}
+    export KUBECTL OPERATOR_IMAGE_NAME
+    ./tests/e2e.sh deploy-operator
+  openshift_installer_src:
+    cluster_profile: aws
```
Comment (petr-muller) [inline ci-operator/config/open-cluster-management/observatorium-operator/open-cluster-management-observatorium-operator-release-2.1.yaml:31]: `openshift_installer_src` is legacy and new jobs that use it should not be added. Multi-stage test wofkflows should be used instead. Please see https://docs.ci.openshift.org/docs/how-tos/migrating-template-jobs-to-multistage/#openshift_installer_src for information about how to write an equivalent in the newer mechanism, and https://docs.ci.openshift.org/docs/architecture/step-registry/ for  general information about multi-stage tests.

File: core-services/template-deprecation/_allowlist.yaml [hunk 3]
```diff
@@ -3235,6 +3235,9 @@ templates:
         pull-ci-ComplianceAsCode-content-master-e2e-aws-rhcos4-moderate:
           generated: false
           kind: presubmit
+        pull-ci-open-cluster-management-observatorium-operator-release-2.1-test:
```
Comment (droslean) [inline core-services/template-deprecation/_allowlist.yaml:3238]: That test doesn't exist, also you don't seem to use any template job as well.

Comment (clyang82) [reply core-services/template-deprecation/_allowlist.yaml]: I have removed it.

**Comment by clyang82** [comment]
@droslean Please have a look if that works. Thanks.

**Review by droslean — APPROVED** [review]

**Pull request merged**